radanalyticsio/spark/cluster/base/role.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: edit-resources
```

Comment:
Can you add `spark` to the name to show that this was created for the spark operator deployment?